res/css/views/settings/tabs/user/_StyleUserSettingsTab.scss

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
This file should be `_AppearanceUserSettingsTab.scss`, it also doesn't seem to be imported in the `_components.scss` index.